tests/test_subgraphs.py

```diff
@@ -292,7 +292,7 @@ def test_scan_for_product(op, onnx_helper):
 
 def test_sequence_map_for_zip_mul(op, onnx_helper):
     xs, ys = arguments(
-        xs=Sequence(Tensor(numpy.int32)), ys=Sequence(Tensor(numpy.int32))
+        xs=Sequence(Tensor(numpy.int64)), ys=Sequence(Tensor(numpy.int64))
```

Comment:
Interesting! How did that work before?

Reply:
Welp, I guess it was just unchecked or there was an implicit cast done. Who knows!
The exact reason in case it's not obvious is that afterwards the arrays are `int64` implicitly.